### ISSUE TITLE
Implement unit testing

### ISSUE THREAD

**Issue by gbasood** [d3athrow/vgstation13#7949]
Ala https://github.com/Baystation12/Baystation12/pull/11808

**Comment by gbasood** [comment]
![](http://i.imgur.com/Tey1hw3.png)
oh no what have I done

**Comment by 9600bauds** [comment]
Explain this in layman's terms for the technologically challenged here

**Comment by PJB3005** [comment]
@9600bauds essentially Travis starts a server and does some basic checks to make sure everything's in order.

**Comment by gbasood** [comment]
Unit testing also means being able to check if your code is functioning properly without directly testing it.

Not sure if we get that level of usefulness out of unit tests on BYOND but hey, more testing tools is never bad
